chain/client/tests/process_blocks.rs

```diff
@@ -2312,39 +2312,62 @@ fn test_query_final_state() {
     assert!(account_state1.amount < TESTING_INIT_BALANCE - TESTING_INIT_STAKE);
 }
 
+#[test]
+fn test_fork_receipt_ids() {
+    let (mut env, tx_hash) = prepare_env_with_transaction();
+
+    let produced_block = env.clients[0].produce_block(1).unwrap().unwrap();
+    env.process_block(0, produced_block.clone(), Provenance::PRODUCED);
```

Comment:
```suggestion
    env.produce_block(0, 1);
```

Reply:
We still need height of said block down the code